hapi/hedera-protobufs/services/consensus_update_topic.proto

```diff
@@ -87,4 +88,58 @@ message ConsensusUpdateTopicTransactionBody {
      * If unspecified, no change.
      */
     AccountID autoRenewAccount = 9;
+
+    /**
+     * Access control for update/delete of custom fees.<br/>
+     * <p>
+     * If set, subsequent consensus_update_topic transactions signed with this
+     *  key MAY update or delete the custom fees for this topic.<br/>
+     * If set to a Key containing a non-empty KeyList, the previous key and new
+     *  key MUST both sign this transaction.<br/>
```

Comment:
Might be better here to state `If set to a valid Key` instead of specifically a non-empty `KeyList`.